celery/app/defaults.py

```diff
@@ -325,6 +325,7 @@ def __repr__(self):
         pool_restarts=Option(False, type='bool'),
         proc_alive_timeout=Option(4.0, type='float'),
         prefetch_multiplier=Option(4, type='int'),
+        enable_prefetch_count=Option(True, type='bool'),
```

Comment:
This is not a meaningful name for the feature flag.
Prefetching is always enabled.
What this feature flag controls is the reduction of the prefetch count when the connection restarts.

Reply:
Can you offer a better name @thedrow ?

Reply:
Maybe `enable_prefetch_count_reduction_on_connection_restart`?

Reply:
`enable_prefetch_count_reduction_on_connection_restart` was too long. Went for:
`enable_prefetch_count_reduction`